ui-tests/src/test/java/org/openmrs/reference/page/VisitTypeListPage.java

```diff
@@ -45,6 +45,7 @@ public VisitTypePage goToVisitType(String name) {
 
     public List<String> getVisitTypeList() {
         List<String> visitTypeList = new ArrayList<String>();
+        driver.navigate().refresh();
```

Comment:
any reason why u want to refresh the page here?

Reply:
The element goes Into stale condition; either the element is read s if its no longer attached to the DOM, or Its read as if it is not in the current frame context. Refreshing the page makes the element accessible

Reply:
Are u only able to resolve that this way only?

Reply:
A smart way for the resolution is in the incoming push.